### PR TITLE
First stab at Vocabularies

### PR DESCRIPTION
**Pull request by maparent**
Not meant to be definitive, but getting comments at this point.

### CODE REVIEW

File: assembl/alembic/versions/95750e0267d8_extract_taxonomies.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+"""extract taxonomies
+
+Revision ID: 95750e0267d8
+Revises: cc4266c17ceb
+Create Date: 2018-02-06 09:06:58.932766
+
```
Comment (vincentfretin) [inline assembl/alembic/versions/95750e0267d8_extract_taxonomies.py]: I just added a alembic step on develop, you will need to rebase and use Revises: ae1da08a88b0

File: assembl/scripts/db_manage.py [hunk 2]
```diff
@@ -33,10 +33,11 @@ def main():
...
     if args.command == "bootstrap":
-        db = bootstrap_db(args.configuration)
-        bootstrap_db_data(db)
-        mark_changed()
-        transaction.commit()
+        with transaction.manager:
+            db = bootstrap_db(args.configuration)
+            mark_changed()
+        with transaction.manager:
+            bootstrap_db_data(db)
```
Comment (vincentfretin) [inline assembl/scripts/db_manage.py]: Is there a reason you do two commits. If yes, can you please add the reason in a comment.

**Comment by vincentfretin** [comment]
The changes looks good.

Once this is merged I will work on indexing nature and action associated to post via extract in elasticsearch

**Comment by maparent** [comment]
Done.

**Pull request merged**